qt/python/mantidqt/mantidqt/widgets/sliceviewer/models/roi.py

```diff
@@ -198,6 +220,25 @@ def _extract_roi_numeric_axis(
     return _extract_region(workspace, xmin, xmax, indexmin, indexmax, roi_name, log_algorithm_calls)
 
 
+def _adjust_xmin_xmax_for_event_workspace(workspace, xmin, xmax, ws_index_min):
```

Comment:
Would it be possible to add a test for this new function?

Reply:
Test file added.